jfxtras-controls/src/main/java/jfxtras/internal/scene/control/skin/CalendarTextFieldSkin.java

```diff
@@ -365,7 +389,7 @@ public String getUserAgentStylesheet() {
 		};
 		lBorderPane.getStyleClass().add(this.getClass().getSimpleName() + "_popup");
 		lBorderPane.setCenter(calendarPicker);
-		calendarPicker.showTimeProperty().set( getSkinnable().getShowTime() );
+		calendarPicker.showTimeProperty().bind(getSkinnable().showTimeProperty());
```

Comment:
I think I did this on purpose; binding caused memory leaking I believe. The picker is only used during the popup and should be destroyed after closing to save memory.